### DISCUSSION TITLE
About GPLv2with the GNU Classpath Exception License

### DISCUSSION THREAD

**Discussion by Ericdmb** [eclipse-theia/theia#13468] [Q&A] [answered]
If my extension dependencies include @theia/core does that mean my extension should also comply with the GPLv2 open source license?

**Comment by msujew** [comment, marked as answer]
Hey @Ericdmb,

no, that is not necessary. Only if your code is a fork/copy of the `theia` repository and you modify the code, you'll be required to open source your modifications. Any adopter of Theia can use any (proprietary/open source) license they want for their own code that uses Theia as a dependency.

**Comment by Ericdmb** [comment]
thank you very much

> Hey @Ericdmb,
> 
> no, that is not necessary. Only if your code is a fork/copy of the `theia` repository and you modify the code, you'll be required to open source your modifications. Any adopter of Theia can use any (proprietary/open source) license they want for their own code that uses Theia as a dependency.
